bin/pullRawDataFromDS.sh

```diff
@@ -257,8 +257,31 @@ else
 	# Cleanup old data if data transfer with rsync finished successfully (and hence did not crash this script).
 	#
 	log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "Deleting data older than 14 days from ${HOSTNAME_DATA_STAGING%%.*}:/groups/${GROUP}/${SCR_LFS}/ ..."
-	# shellcheck disable=SC2029
-	/usr/bin/ssh "${HOSTNAME_DATA_STAGING}" "/bin/find /groups/${GROUP}/${SCR_LFS}/ -mtime +14 -ignore_readdir_race -delete"
+	## get the project name by parsing the /home/umcg-ndewater/files/ folder, directories only and no empty or '.'
+	readarray -t gsProjectsSourceServer< <(rsync -f"+ */" -f"- *" "${HOSTNAME_DATA_STAGING}:/home/umcg-ndewater/files/" | awk '{if ($5 != "" && $5 != "."){print $5}}')
```

Comment:
"Project" is verwarrend hier. Het gaat om batches en die kunnen meerdere projecten bevatten.